tests/test_helper/microovn.bash

```diff
@@ -294,3 +314,83 @@ function wait_microovn_online() {
 
     return $rc
 }
+
+# wait_ovsdb_cluster_changes_applied CONTAINER CONTROL_PATH DB_NAME TIMEOUT
+#
+# Wait until OVN OVSDB cluster member converges with rest of the cluster. This function
+# checks output of ovn-appctl to make sure that field 'Entries not yet applied' reaches 0.
+# It requires CONTROL_PATH which points to database's .ctl file (i.e. path/to/ovnnb_db.ctl)
+# and DB_NAME which should be either "OVN_Northbound" or "OVN_Southbound.
+#
+# TIMEOUT in seconds is roughly obeyed. If conditions are not met before timeout is reached, this
+# functions returns non-zero RC
+function wait_ovsdb_cluster_changes_applied() {
+    local container=$1; shift
+    local ctl_path=$1; shift
+    local db_name=$1; shift
+    local timeout=$1; shift
+    local rc=1
+    local retries=""
+    retries=$((timeout * 2))
+
+    for (( i = 1; i <= "$retries"; i++ )); do
+        echo "# ($container) Waiting for $db_name to apply all changes ($i/$retries)"
+        run lxc_exec "$container" "microovn.ovn-appctl -t $ctl_path cluster/status $db_name"
```

Comment:
nit: there is already a function for retrieving the cluster status here:
https://github.com/canonical/microovn/blob/2a0e6b556a02074774dcc5fc50f835a8f2165b50/tests/test_helper/microovn.bash#L148-L150